hledger-mode.el

```diff
@@ -125,15 +130,40 @@ COMMAND, ARG and IGNORED the regular meanings."
 (defun hledger-font-lock-keywords-1 ()
   "Minimal highlighting expressions for hledger mode."
   (list
-   `(,(concat "^  +" hledger-account-regex) 1 hledger-account-face)
-   `(,(concat "^account " hledger-account-regex "$") 1 hledger-account-face)
-   `(,(concat "^alias \\([^[:space:]=;\n]+\\) = " hledger-account-regex "$")
+   `(,(concat "^  +" hledger-account-regex) (1 hledger-account-face)
+     (hledger--fontify-account-colons (hledger--fontify-account-colons-pre) (hledger--fontify-account-colons-post) (0 hledger-account-colon-face t)))
+   `(,(concat "^account " hledger-account-regex "$") (1 hledger-account-face)
+     (hledger--fontify-account-colons (hledger--fontify-account-colons-pre) (hledger--fontify-account-colons-post) (0 hledger-account-colon-face t)))
+   `(,(concat "^payee " hledger-account-regex "$") (1 hledger-account-face)
```

Comment:
I am trying to wrap my head around the format used here. Is this the format used here?
```emacs-lisp
(MATCHER HIGHLIGHTER1 HIGHLIGHER2)
```
where `HIGHLIGHTER 1` is a simple higlighter using captured group #1in the `MATCHER` regexp
and `HIGHLIGHTER 2` is an `ANCHORED HIGHLIGHTER` using the three functions to guide the search for `:` chars.

Reply:
Yup, that’s it. It turned out to be a bit complicated.